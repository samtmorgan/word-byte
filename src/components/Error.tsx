import React from 'react';

export const Error = () => <div>ğŸ˜­ Ouch! Something went wrong, please try again.</div>;
