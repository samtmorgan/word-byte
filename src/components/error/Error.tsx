import React from 'react';

export default function ErrorPage() {
  return (
    <div className="page-container">
      <h1>ðŸ˜­ Ouch! Something went wrong, please try again.</h1>
    </div>
  );
}
