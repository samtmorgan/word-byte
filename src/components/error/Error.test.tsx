import React from 'react';
import { render } from '@testing-library/react';
import Error from './Error';

describe('Loader component', () => {
  test('renders loading text', () => {
    const { container } = render(<Error />);

    expect(container).toMatchSnapshot();
  });
});
