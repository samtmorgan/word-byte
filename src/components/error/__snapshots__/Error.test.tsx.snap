// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component renders loading text 1`] = `
<div>
  <div
    class="pageContainer"
  >
    <h1>
      ðŸ˜­ Ouch! Something went wrong, please try again.
    </h1>
  </div>
</div>
`;
